.github/workflows/_jobs_ci.yml

```diff
@@ -18,71 +18,43 @@ jobs:
       - name: "Clone repository"
         uses: "actions/checkout@v3.4.0"
 
-      - name: "Prepare Repository"
-        # Mark the repository as a safe directory in git: https://github.com/actions/checkout/issues/766
-        run: "git config --global --add safe.directory $GITHUB_WORKSPACE"
-
       - name: "Restore CI cache"
         if: "${{ github.ref_name != 'main' }}"
         uses: "actions/cache/restore@v3.3.1"
         with:
-          key: "ci-cache-${{ hashFiles('**/Cargo.lock', '**/package-lock.json', '**/Pipfile.lock') }}"
-          restore-keys: "ci-cache-"
+          key: "cache-${{ github.ref_name }}-${{ hashFiles('Cargo.lock') }}-${{ hashFiles('package-lock.json') }}"
+          restore-keys: |
+            cache-${{ github.ref_name }}-
+            cache-main-
+            cache-
           path: |
-            **/.hermit/
-            **/node_modules/
-            **/target/
-
-      - name: "Build devcontainer"
-        uses: "./.devcontainer"
-        with:
-          entrypoint: "echo"
+            .hermit/rust/bin/
+            .hermit/rust/git/db/
+            .hermit/rust/registry/cache/
+            .hermit/rust/registry/index/
+            .hermit/sccache/
 
       #
       # Run all CI steps in order: _SLANG_INFRA_CI_STEPS_ORDERED_ (keep in sync)
       #
 
-      - name: "scripts/bin/infra setup"
-        uses: "./.devcontainer"
-        with:
-          entrypoint: "scripts/bin/infra"
-        env:
-          SLANG_INFRA_CLI_EXTRA_ARGS: "setup"
-
-      - name: "scripts/bin/infra check"
-        uses: "./.devcontainer"
-        with:
-          entrypoint: "scripts/bin/infra"
-        env:
-          SLANG_INFRA_CLI_EXTRA_ARGS: "check"
-
-      - name: "scripts/bin/infra test"
-        uses: "./.devcontainer"
-        with:
-          entrypoint: "scripts/bin/infra"
-        env:
-          SLANG_INFRA_CLI_EXTRA_ARGS: "test"
-
-      - name: "scripts/bin/infra lint"
-        uses: "./.devcontainer"
-        with:
-          entrypoint: "scripts/bin/infra"
-        env:
-          SLANG_INFRA_CLI_EXTRA_ARGS: "lint"
+      - run: "scripts/bin/infra setup"
```

Comment:
Shouldn't we use `infra ci` here? Would help us drop one "keep in sync" marker.

Reply:
The issue with that is GitHub web log viewer hangs up on really large output :(
Otherwise, I would need to download the raw logs zip file and grep through it locally.

Reply:
Ah, gotcha. I understand that using collapsible/"step" GH markers won't help us here?

Reply:
Unfortunately collapsible groups are checked/collapsed client side, and only after the whole group is parsed/rendered.  It works great when viewing a completed job, but doesn't help with streaming logs :(